tests/python/unittest/test_image.py

```diff
@@ -118,6 +118,22 @@ def test_resize_short(self):
                     mx_resized = mx.image.resize_short(mx_img, new_size, interp)
                     assert_almost_equal(mx_resized.asnumpy()[:, :, (2, 1, 0)], cv_resized, atol=3)
 
+    def test_imresize(self):
+        try:
+            import cv2
+        except ImportError:
+            return
```

Comment:
Throw a skip test exception please

Reply:
will add `raise unittest.SkipTest("Unable to import cv2.")`